docs/preset-env.md

```diff
@@ -229,38 +229,69 @@ This option is useful for "blacklisting" a transform like `@babel/plugin-transfo
 
 `"usage"` | `"entry"` | `false`, defaults to `false`.
 
-> This option adds direct references to the `core-js` module as bare imports. Thus `core-js` will be resolved relative to the file itself and needs to be accessible. You may need to specify `core-js@2` as a top level dependency in your application if there isn't a `core-js` dependency or there are multiple versions.
-
 This option configures how `@babel/preset-env` handles polyfills.
 
+When either the `usage` or `entry` options are used, `@babel-preset-env` will add direct references to `core-js` modules as bare imports (or requires). This means `core-js` will be resolved relative to the file itself and needs to be accessible.
+
```

Comment:
The same as in `runtime` docs - I don't see any links to `core-js` documentation in this file.

Reply:
`Built-ins` link in `include` should be updated.